pkgs/development/python-modules/telfhash/default.nix

```
@@ -17,6 +17,8 @@ buildPythonPackage {
     sha256 = "jNu6qm8Q/UyJVaCqwFOPX02xAR5DwvCK3PaH6Fvmakk=";
   };
 
+  patches = [ ./telfhash-new-tlsh-hash.patch ];
```

Maybe we should add an argument like `enableNewHash` which defaults to true, so that you can use this with both old and new hashes?

Then this would become:

```suggestion
  patches = lib.optionals enableNewHash [
    # enable support for new TLSH has format
    ./telfhash-new-tlsh-hash.patch
  ];
```

---

This would work, but only as soon as https://github.com/trendmicro/tlsh/issues/101 has been fixed.

---

So, right now it's broken for both old and new hashes?

---

Yes. To be more specific, the "grouping" feature is broken for both the old and new format hash. The patches in this PR fix it for the new format hash.

---

Got it, I understand now. Thanks for the clarification. Then, I'd say this is good to go.